components/admin/lessons/AdminLessonNav.tsx

```diff
@@ -0,0 +1,80 @@
+import { toUpper } from 'lodash'
+import { uniq } from 'lodash/fp'
+import React, { useState, useMemo } from 'react'
+import { Tab, Col, Nav } from 'react-bootstrap'
+import styles from '../../../scss/adminLessonNav.module.scss'
+
+type NavItem = { value: string }
+type Tab = () => JSX.Element
+
+type NavItems = [NavItem, ...NavItem[]]
+type TabsType = [Tab, ...Tab[]]
+
+type Props = {
```

Comment:
What was wrong with the previous type definition for props?
```
type AdminLessonNavProps = {
  navItems: { value: string }[]
  tabs: (() => JSX.Element)[]
}
```

Reply:
The current one doesn't allow empty arrays to be passed.